src/Main.ts

```diff
@@ -1,148 +1,132 @@
+import { Logger, LogLevel } from "@itwin/core-bentley/lib/cjs/Logger";
+import { LocalBriefcaseProps, BriefcaseIdValue } from "@itwin/core-common/lib/cjs/BriefcaseTypes";
+import { IModelVersion } from "@itwin/core-common";
+import { ProgressFunction } from "@itwin/core-backend";
+import { Readline } from "readline/promises";
+
+// Find your iTwin and iModel IDs at https://developer.bentley.com/my-imodels/
+const IMODELHUB_REQUEST_PROPS = {
+  iTwinId: "e1af3a89-b637-4cea-91a9-1f38789eee33", // EDIT ME! Specify your own iTwinId
+  iModelId: "ea48f654-987e-44b3-bdb6-5aeb36d15f33", // EDIT ME! Specify your own iModelId
+};
+
+const AUTH_CLIENT_CONFIG_PROPS = {
+  clientId: "native-uBy8v6uCZ8QZfcFn1CecLrnBD", // EDIT ME! Specify your own clientId
+  /** These are the minimum scopes needed - you can leave alone or replace with your own entries */
+  scope: "imodels:read",
+  /** This can be left as-is assuming you've followed the instructions in README.md when registering your application */
+  redirectUri: "http://localhost:3000/signin-callback",
+};
+
+const APP_LOGGER_CATEGORY = "imodel-report-main";
+
+(async () => {
+  const imhConfig: IModelHostConfiguration = {
+    hubAccess: new BackendIModelsAccess(), // needed to download iModels from iModelHub
+    // These tile properties are unused by this application, but are required fields of IModelHostConfiguration.
+    logTileLoadTimeThreshold: IModelHostConfiguration.defaultLogTileLoadTimeThreshold,
+    logTileSizeThreshold: IModelHostConfiguration.defaultLogTileSizeThreshold,
+    tileContentRequestTimeout: IModelHostConfiguration.defaultTileRequestTimeout,
+    tileTreeRequestTimeout: IModelHostConfiguration.defaultTileRequestTimeout,
+  };
+  await IModelHost.startup(imhConfig);
 
-import readline = require("readline");
+  // await deletePassword(`Bentley.iTwinJs.OidcTokenStore.${AUTH_CLIENT_CONFIG_PROPS.clientId}`, "Glen.Worrall");
 
-async function signIn(): Promise<AccessToken> {
-  const config: NativeAppAuthorizationConfiguration = {
-    clientId: "imodeljs-electron-samples",
-    redirectUri: "http://localhost:3000/signin-callback",
-    scope: "openid email profile organization imodelhub context-registry-service:read-only product-settings-service urlps-third-party offline_access",
-  };
+  Logger.initializeToConsole();
+  Logger.setLevel(APP_LOGGER_CATEGORY, LogLevel.Info);
 
-  const client = new ElectronAuthorizationBackend();
-  await client.initialize(config);
-
-  return new Promise<AccessToken>((resolve, reject) => {
-    NativeHost.onUserStateChanged.addListener((token) => {
-      if (token !== undefined) {
-        resolve(token);
-      } else {
-        reject(new Error("Failed to sign in"));
-      }
-    });
-    client.signIn().catch((err) => reject(err));
-  });
-}
+  let userdata;
+  const json = process.argv[2];
+  if (json === undefined) {
+    userdata = require("../queries/ifc.json");
```

Comment:
The code on main loads example.json, we should continue to load that file instead of the ifc one